src/patterns/dosdp-patterns/blood_vessel_pattern.yaml

```diff
@@ -0,0 +1,78 @@
+pattern_name: blood_vessel_pattern
+pattern_iri: http://purl.obolibrary.org/obo/uberon/patterns/blood_vessel_pattern.yaml
+description: "This is a pattern for a blood vessel."
+
+
+classes:
+  blood vessel: "UBERON:0001981"
+
+relations:
+  connecting branch of: "RO:0002252"
+  tributary of: "RO:0002376"
+  vessel supplies blood to: "RO:0020101"
+  vessel drains blood from: "RO:0020102"
+  part of: "RO:0000050"
+
+annotationProperties:
+  obo_foundry_uniquename: "http://purl.obolibrary.org/obo/IAO_0000589"
+  hasExactSynonym: "oboInOwl:hasExactSynonym"
+  hasDbXref: "oboInOwl:hasDbXref"
+
+vars:
+  parent: ""
+  connecting_branch_of: ""
+  tributary_of: ""
+  vessel_supplies_blood_to: ""
+  vessel_drains_blood_from: ""
+  part_of: ""
+  creator: ""
+
+data_list_vars:
+  synonym: ""
+  xrefs: ""
+  synonym_xrefs: ""
+
+data_vars:
+  label: ""
+  definition: ""
+
+name:
+  text: "%s"
+  vars:
+    - label
+
+def:
+  text: "%s"
+  vars:
+    - definition
+  xrefs: xrefs
+
+exact_synonym:
+  value: synonym
+  xrefs: synonym_xrefs
+  
+logical_axioms:
+  - axiom_type: subClassOf
+    text: "%s"
+    vars:
+      - parent
+  - axiom_type: subClassOf
+    text: "'part of' some %s"
+    vars:
+      - part_of
+  - axiom_type: subClassOf
+    text: "'connecting branch of' some %s"
+    vars:
+      - connecting_branch_of
+  - axiom_type: subClassOf
+    text: "'tributary of' some %s"
+    vars:
+      - tributary_of
+  - axiom_type: subClassOf
+    text: "'vessel supplies blood to' some %s"
+    vars:
+      - vessel_supplies_blood_to
+  - axiom_type: subClassOf
+    text: "'vessel drains blood from' some %s"
+    vars:
+      - vessel_drains_blood_from
```

Comment:
It would be good to add the date and the contributor.

Reply:
Every time I add the contributor fails.

Reply:
But for each term? Did you try

```yaml
annotationProperties:
  contributor: "http://purl.org/dc/terms/contributor"
  date: "http://purl.org/dc/terms/date"
annotations:
  - annotationProperty: contributor
    var: creator
  - annotationProperty: date
    text: %s
    var: date_created
```
?